content/organization/tech/security/index.en.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Security"
+menuTitle: "Security"
+weight: 54
+---
+
+As organizers you will be responsible for accounts such as email, social media accounts.
+Here are some steps that will help reduce the risk of someone's hacking into your account.
+
+## Use a personal password manager!
+
+For your personal use (for R-Ladies things and in general!), we strongly recommend using a personal password manager, be it a free one like KeePass (with a back-up you'd set up on Dropbox or Google Drive) or a paid service like 1Password.
```

Comment:
For your personal use (for R-Ladies things and in general!), we strongly recommend using a personal password manager, be it a free one like KeePass (with a back-up you'd set up on Dropbox or Google Drive) or [Bitwarden](https://bitwarden.com/), or a paid service like [1Password](https://1password.com/).